docs/cf_remote_demo.sh

```diff
@@ -0,0 +1,37 @@
+
+set -e
+set -x
+
+cf-remote spawn --platform ubuntu-18-04-x64 --count 1 --name hub --role hub
+cf-remote spawn --platform ubuntu-18-04-x64 --count 1 --name ubuntu18 --role client
+cf-remote spawn --platform ubuntu-16-04-x64 --count 1 --name ubuntu16 --role client
+cf-remote spawn --platform ubuntu-14-04-x64 --count 1 --name ubuntu14 --role client
+cf-remote spawn --platform centos-6-x64 --count 1 --name centos6 --role client
+cf-remote spawn --platform centos-7-x64 --count 1 --name centos7 --role client
+cf-remote spawn --platform debian-7-x64 --count 1 --name debian7 --role client
+cf-remote spawn --platform debian-8-x64 --count 1 --name debian8 --role client
+cf-remote spawn --platform debian-9-x64 --count 1 --name debian9 --role client
+cf-remote spawn --platform rhel-5-x64 --count 1 --name rhel5 --role client
+cf-remote spawn --platform rhel-6-x64 --count 1 --name rhel6 --role client
+cf-remote spawn --platform rhel-7-x64 --count 1 --name rhel7 --role client
+
+cf-remote --version master install --demo --bootstrap hub --hub hub
+
+cf-remote deploy --hub hub /northern.tech/cfengine/masterfiles
+
+cf-remote --version master install --demo --bootstrap hub --clients ubuntu18
+cf-remote --version master install --demo --bootstrap hub --clients ubuntu16
+cf-remote --version master install --demo --bootstrap hub --clients ubuntu14
+cf-remote --version master install --demo --bootstrap hub --clients centos6
+cf-remote --version master install --demo --bootstrap hub --clients centos7
+cf-remote --version master install --demo --bootstrap hub --clients debian7
+cf-remote --version master install --demo --bootstrap hub --clients debian8
+cf-remote --version master install --demo --bootstrap hub --clients debian9
+cf-remote --version master install --demo --bootstrap hub --clients rhel5
+cf-remote --version master install --demo --bootstrap hub --clients rhel6
+cf-remote --version master install --demo --bootstrap hub --clients rhel7
```

Comment:
Hmm, would `ubuntu18,ubuntu16,ubuntu14,...` work?

Reply:
Yes, but I find it handy that it's easy to comment out lines.